api/src/middlewares/mongodb-sanitizer.test.ts

```diff
@@ -0,0 +1,82 @@
+import { mongodbSanitizer } from './mongodb-sanitizer';
```

Comment:
Can you add also an example with the following:
{
   "key": null
}
Just to be sure that there is not problem in null checking

Reply:
added additional test cases 👍